src/formatter.rs

```diff
@@ -85,8 +154,21 @@ pub fn format_cg(text: &str, ostyle: &OutputStyle) -> String {
         })
         .to_string();
 
-    result = RE_VARIABLE
-        .replace_all(&result, |caps: &regex::Captures| ostyle.variable.paint(&caps[1]).to_string())
+    result
+}
+
+/// NOTE: [[VARIABLE]] - {{CONSTANT}} - <<BOLD>> - `MONOSPACE`
+///
+/// Removes formatting tags and paints the inner content accordingly:
+///     [[VARIABLE]]
+///  -> \u{1b}[33mVARIABLE\u{1b}[0m
```

Comment:
The doc comments in rust support markdown which makes these render weird in the hover text:
![image](https://github.com/Andriamanitra/clash/assets/10672443/c08cd54f-0af7-4606-b6dd-b8bafa901634)

Reply:
I will replace most of those for just two //, until we figure out what to do with all these links.